website/docs/cloud-docs/workspaces/settings/drift-detection.mdx

```diff
@@ -8,53 +8,4 @@ tfc_only: true
 
```

Comment:
This is interesting --> Can we just delete this page entirely and link users to the new page? We've got a redirect so that feels reasonable!